### PR TITLE
added seamless config

### PR DESCRIPTION
**Pull request by ephraimm**

### CODE REVIEW

**Comment by ephraimm** [comment]
I signed it!

**Comment by ephraimm** [comment]
I signed it!

File: sources/seamless_fleets.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "SEAMLESS_FLEETS",
+  "name": "Seamless Fleets",
+  "categories": ["DATABASE"],
+  "organization": "SEAMLESS_FLEETS",
+  "iconUrl": "https://www.seamlessfleets.co.za/wp-content/uploads/2018/09/seamless-symbol.png",
+  "sourceUrl": "https://www.seamlessfleets.co.za/",
+  "dataVisibility": ["PUBLIC"]
```
Comment (pfrisella) [inline sources/seamless_fleets.json]: Can you confirm that PUBLIC is the correct choice? See description for dataVisibility at https://github.com/googledatastudio/ds-data-registry#add-a-source

**Comment by pfrisella** [comment]
Closing due to inactivity. If this is still needed send a new pull request and address the comment regarding data visibility.

**Pull request closed without merging**